release-notes.py

```diff
@@ -62,11 +72,30 @@
     print("Writing to RELEASE.md OK!")
     release_suppfile = open("release_supp.data", "a")
     try:
-        release_suppfile.write(app_ver_num+"\n") == 0
+        release_suppfile.write(app_ver_numbr+"\n") == 0
     except IOError:
         print("Error writing to support file!")
     else:
         print("Writing to support file OK!")
     release_suppfile.close()
 
 releasemd.close()
+
+message_text = ""
```

Comment:
Here you can simplify conditions and make it better readable: 
```
# default value
message_text = app_msg+app_ver_numbr
```
then you can use just if condition. Else will be not necessary any more